server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -760,7 +760,18 @@ func (s *BuildBuddyServer) GetUserApiKeys(ctx context.Context, req *akpb.GetApiK
 		return nil, status.UnimplementedError("Not Implemented")
 	}
 	groupID := req.GetRequestContext().GetGroupId()
-	tableKeys, err := authDB.GetUserAPIKeys(ctx, groupID)
+
+	u, err := s.env.GetAuthenticator().AuthenticatedUser(ctx)
+	if err != nil {
+		return nil, err
+	}
+	// TODO: always set req.user_id
```

Comment:
might want to file a bug if you're not going to send a followup right after this?

Reply:
This has to be done over multiple rollouts so I'll file a bug just so I don't forget.